apps/admin/src/components/prompts/standard/submit-prompt.vue

```diff
@@ -1,16 +1,41 @@
 <template>
-  <v-tab-item key="options" value="options"></v-tab-item>
+  <v-tab-item key="options" value="options">
+    <v-card-title>{{ $t('survey-schemes.prompts.submit-prompt.optionTitle') }}</v-card-title>
+    <v-card-subtitle>{{
+      $t('survey-schemes.prompts.submit-prompt.optionSubtitle')
+    }}</v-card-subtitle>
+    <v-card-text>
+      <v-combobox
+        v-for="key in Object.keys(review)"
+        :key="key"
+        :items="[false, 'checkbox', 'scroll']"
+        :label="key"
```

Comment:
`:label="survey-schemes.actions.layouts.${key}"`

Reply:
Thanks. It seems that we need to add $t() around it `:label="$t(`survey-schemes.actions.layouts.${key}`)"`